### PR TITLE
Main window

### PR DESCRIPTION
**Pull request by Artem0904**

### CODE REVIEW

File: FinanceManager/ChangeLimitWindow.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+            LimitTextBox.Text = string.Empty;
+            this.Close();
+        }
+
+        private void Cancel_Click(object sender, RoutedEventArgs e)
+        {
+            this.Close();
+        }
+
```
Comment (vladtymo) [inline FinanceManager/ChangeLimitWindow.xaml.cs:48]: Remove line

File: FinanceManager/ChangeLimitWindow.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        {
+            if(!int.TryParse(LimitTextBox.Text, out NewLimit))
+            { 
+                MessageBox.Show($"The 'limit' field must contain only a numeric value without text or special characters! And limit cannot be greater than {MaxLimit}");
+                LimitTextBox.Text = string.Empty;
+                return;
+            }
+            // запис нового ліміта в базу щоб потім змінювати ліміт в мейн віндов
+
+
```
Comment (vladtymo) [inline FinanceManager/ChangeLimitWindow.xaml.cs:38]: Remove these lines

File: FinanceManager/ResoursesXAML/Styles.xaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        <Setter Property="Background" Value="LightGray"></Setter>
+        <Setter Property="FontSize" Value="16"></Setter>
+    </Style>
+    
+    <Style x:Key="BorderStyle" TargetType="{x:Type Border}">
+        <Setter Property="Background" Value="Black"></Setter>
+        <Setter Property="CornerRadius" Value="7"></Setter>
+        <Setter Property="BorderThickness" Value="1"/>
+        <Setter Property="BorderBrush" Value="White"/>
+
```
Comment (vladtymo) [inline FinanceManager/ResoursesXAML/Styles.xaml:44]: Remove line

**Review by vladtymo — APPROVED** [review]

**Review by PrymakAndrii — COMMENTED** [review]
Perfect

**Pull request merged**